src/compute-pr-actions.ts

```diff
@@ -407,6 +407,8 @@ function createWelcomeComment(info: PrInfo, staleness: Staleness) {
         const infraFiles = info.files.filter(f => f.kind === "infrastructure");
         const links = infraFiles.map(f => `[\`${f.path}\`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/${info.headCommitOid}/${f.path})`);
         display(` * ${emoji(approval.approved)} A DT maintainer needs to approve changes which affect DT infrastructure (${links.join(", ")})`);
+    } else if (info.dangerLevel === "MultiplePackagesEdited") {
+        display(` * ${emoji(approval.approved)} A DT maintainer needs to approve changes which affect more than one package`);

```

Comment:
This used to fall through to the final else which is "without tests" - https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46321 definitely has tests

Reply:
Yeah, that should be added, but see below.